### PR TITLE
Redmine#7098: let filestat() support xattr

### PR DESCRIPTION
**Pull request by kacfengine**
Supersedes #2164.

### CODE REVIEW

**Comment by kacfengine** [comment]
trigger build

**Comment by kacfengine** [comment]
@tzz: Build is green on this one. Care to mention the xattr value in the docs?

File: tests/acceptance/01_vars/02_functions/filestat_xattr.cf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  classes:
+      "has_xattr" and => { fileexists($(xattr)) };
+}
+
+#######################################################
+
+bundle agent init
+{
+  meta:
+      "test_skip_unsupported" string => "!has_xattr";
```
Comment (tzz) [inline tests/acceptance/01_vars/02_functions/filestat_xattr.cf]: aha sweet

**Comment by tzz** [comment]
works for me :)

**Comment by tzz** [comment]
OK, docs in https://github.com/cfengine/documentation/pull/1024

**Comment by kacfengine** [comment]
Great, thanks!

**Pull request merged**